capstone/src/test/java/com/google/sps/objects/TimeTest.java

```diff
@@ -17,7 +17,7 @@
 public class TimeTest {
 
   @Spy private Time mockedTime;
-  private static long EXPECTED_DATE = 1595044799000L;
+  private static long EXPECTED_DATE = 1595030399000L;
```

Comment:
I think at least someone else has a different value for the expected date so these two keep changing on every PR lol. I'm just gonna ask you to keep the old one so it doesn't always change

Reply:
Yeah we're not really sure why it keeps changing, but we keep have to switching so tests don't fail. I'll switch it back though!